bin/kafo-export-params

```diff
@@ -31,7 +31,7 @@ module Kafo
       c                         = Configuration.new(config, false)
       KafoConfigure.config      = c
       KafoConfigure.root_dir    = File.expand_path(c.app[:installer_dir])
-      KafoConfigure.module_dirs = File.expand_path(c.app[:module_dirs])
+      KafoConfigure.module_dirs = c.app[:module_dirs].map { |path| File.expand_path(path) }
```

Comment:
If module_dirs is set to a string (the default with config/kafo.yaml.example), this throws an error as the app config's simply a string.  Configuration uses `[app[:module_dirs]].flatten`, perhaps adopt this?  Or even better, can you just call `c.module_dirs` here?